src/test/fuzz/script.cpp

```diff
@@ -56,22 +56,43 @@ FUZZ_TARGET_INIT(script, initialize_script)
     }
 
     CTxDestination address;
-    (void)ExtractDestination(script, address);
-
     TxoutType type_ret;
     std::vector<CTxDestination> addresses;
     int required_ret;
-    (void)ExtractDestinations(script, type_ret, addresses, required_ret);
-
-    const FlatSigningProvider signing_provider;
-    (void)InferDescriptor(script, signing_provider);
-
-    (void)IsSegWitOutput(signing_provider, script);
-
-    (void)IsSolvable(signing_provider, script);
+    if (!ExtractDestinations(script, type_ret, addresses, required_ret)) {
```

Comment:
This code could be significantly more efficient by just calling `ExtractDestination`(`s`) once each, caching the result, and then making assertions on those results.